_includes/head.html

```diff
@@ -34,6 +34,10 @@
     <script src="/javascripts/jquery.loadTemplate-1.4.4.min.js"></script>
     <script src="/javascripts/respond.js"></script>
     <script src="/javascripts/ua-parser.min.js"></script>
+    <script src="https://npmcdn.com/imagesloaded@4.1/imagesloaded.pkgd.min.js"></script>
```

Comment:
This helper is called even if the jQuery set it is given is empty with my current usage. I did it that way because a) this script is useful for other things and b) I don't want to have to manage/load another 3-line script file for one page. It's this code that uses it:

```js
$('#project-thumbnail-grid').imagesLoaded().progress(function() {
    //...
}
```